.travis.yml

```diff
@@ -32,8 +32,10 @@ env:
   global:
     - MONGODB_3_4=3.4.17
     - MONGODB_3_6=3.6.12
+    - PYMONGO_3_X=3.x
```

Comment:
Now that we're touching this code again, I think it's the right time to make it less confusing...

The problem here is that it's not immediately clear what "PyMongo 3.x" means. Is it 3.6.x? 3.8.x? And If a new PyMongo v3.x.y is released, will our code automatically use that version? Looking at this code, I'd expect so (which would have its own issues – our test suite could break because of changes unrelated to our code). However, that is not really the case because of how `tox.ini` defines `mg3x`: https://github.com/MongoEngine/mongoengine/blob/00e04774845a6c0ca9f1a89d03bb9280af78fca0/tox.ini#L10

I propose we:
1. Change `PYMONGO_3_x=3.x` to `PYMONGO_3_6=3.6`
2. Change `tox.ini`'s `mg3x: PyMongo>=3.0,<3.7` to `mg36: pymongo>=3.6,<3.7` (note the casing difference – after all the PyPI package name is [pymongo](https://pypi.org/project/pymongo/)).
3. In a follow-up PR, add `PYMONGO_3_8` and try to test our latest supported MongoDB/PyMongo/Python job against it. If it works, great. If not, open an issue about supporting PyMongo v3.8.

How does that sound @bagerard?

Reply:
sure